cyclonedx_py/_internal/utils/mimetypes.py

```diff
@@ -34,6 +34,7 @@
     '.license': _MIME_TEXT_PLAIN,
     '.licence': _MIME_TEXT_PLAIN,
     # add more mime types. pull-requests welcome!
+    '.rtf': 'application/rtf',  # `text/rtf` could be used, but application fits better
```

Comment:
```suggestion
    '.rtf': 'application/rtf',
```
https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/MIME_types/Common_types
> .rtf | Rich Text Format (RTF) | application/rtf